client/src/lib/utils/networkData.ts

```diff
@@ -58,6 +69,7 @@ export const Trappist: NetworkData = {
 
 export const Networks: { network: NetworkData; url: string }[] = [
   { network: Rococo, url: (base as string) || "/" },
+  { network: Paseo, url: (base as string) || "/paseo" },
```

Comment:
Shouldn't it be the following?

```suggestion
  { network: Paseo, url: `${base as string}/paseo`},
```